bib/cmd/bootc-image-builder/main.go

```diff
@@ -638,6 +637,17 @@ func run() error {
 	}
 	buildCmd.MarkFlagsRequiredTogether("aws-region", "aws-bucket", "aws-ami-name")
 
+	// If no subcommand is given, assume the user wants to use the build subcommand
+	// See https://github.com/spf13/cobra/issues/823#issuecomment-870027246
+	// which cannot be used verbatim because the arguments for "build" like
+	// "quay.io" will create an "err != nil". Ideally we could check err
+	// for something like cobra.UnknownCommandError but cobra just gives
+	// us an error string
+	_, _, err := rootCmd.Find(os.Args[1:])
```

Comment:
If https://github.com/spf13/cobra/pull/2178 gets accepted we can do the check below in a more precise way, i.e. something like `if _, ok := err.(cobra.UnknownCommandError); ok {`

Reply:
Right, thanks for double-checking this! :) Let's get this merged?